storage/lmdb/src/lib.rs

```diff
@@ -176,6 +176,9 @@ impl backend::Backend for Lmdb {
     type Impl = LmdbImpl;
 
     fn open(self, desc: DbDesc) -> storage_core::Result<Self::Impl> {
+        // Attempt to create the storage directry
+        std::fs::create_dir_all(&self.path).map_err(error::process_io_error)?;
```

Comment:
The policy for creating directories should be handled with care. The default policy I see reasonable (same one as bitcoin's) is that:
- If the data directory is created by default in the default path, then create everything without question
- If the data directory is chosen manually, it has to exist and nothing should be created.

This is probably not directly related to lmdb. This has to be handled in the node runner. It just occurred to me now.